### DISCUSSION TITLE
so, can this run graphicless?(console only)

### DISCUSSION THREAD

**Discussion by Darkar25** [iceiix/stevenarella#534] [Q&A] [answered]
can i run this client from console, without graphics?i want to open multiple minecraft sessions, but if i try to do this with real minecraft, my computer will blow up ;)
I need only Forge support and Chat (and of course walking and world interaction)

**Comment by iceiix** [comment, marked as answer]
In theory the protocol implementation in a separate crate (steven_protocol) can be used standalone, but code would have to be written to do this, it is not a focus of this project currently (though I admit would be useful if someone wants to take it up, at least as a protocol test client in Rust, could be useful to use independent from the graphical client).

This other project may be more useful for your purposes: https://github.com/ORelio/Minecraft-Console-Client/
or, for more advanced custom bots: https://github.com/PrismarineJS/mineflayer
